src/main/java/gift/repository/WishRepository.java

```diff
@@ -1,73 +1,17 @@
 package gift.repository;
 
-import gift.dto.WishRequestDto;
+import gift.entity.Member;
+import gift.entity.Product;
 import gift.entity.Wish;
 import java.util.List;
-import org.springframework.jdbc.core.RowMapper;
-import org.springframework.jdbc.core.simple.JdbcClient;
-import org.springframework.jdbc.support.GeneratedKeyHolder;
-import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public class WishRepository {
-    private final JdbcClient jdbcClient;
+public interface WishRepository extends JpaRepository<Wish, Long> {
+    public List<Wish> findByMember(Member member);
```

Comment:
WishRepository가 interface이기 때문에 여기에 지정해준 접근제어자는 불필요할 것 같네요. 
```suggestion
    List<Wish> findByMember(Member member);
```